addons/skin.estuary/xml/Variables.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <includes>
+        <variable name="SeasonEpisode">
+	        <value>$INFO[ListItem.Season,,[COLOR grey]x[/COLOR]]$INFO[ListItem.Episode]</value>
```

Comment:
OK I tried reverting to a scaled back version without using variables, but things get very complicated when you consider that EpisodeName, Season and Episode are all optional fields and you need a space separating Title or EpisodeName from the 'SeasonEpisode' combination.

Without a variable you have to do odd things which leave a trailing space in the list at all times and make the xml code hard to understand / maintain (something @ronie pointed out against one of my previous attempts)
<code>$INFO[ListItem.Label,, ]$INFO[ListItem.EpisodeName,(,) ]$INFO[ListItem.Season,,x]$INFO[ListItem.Episode,,]</code>

Otherwise you end up with no space shown between the title and episode for specials
HumansS1
Humans 1x2
Battlestar Galactical (Razor)S2
Battlestar Galactical (33) 1x1

Using a variable makes the code easier to read and the spacing just works:
<code><label>$INFO[ListItem.Label]$INFO[ListItem.EpisodeName, (,)]$VAR[SeasonEpisode, ,]</label></code>

Humans S1
Humans 1x2
Battlestar Galactical (Razor) S2
Battlestar Galactical (33) 1x1

I'm not really a skinner, so if there's a better way to do this without using variables please educate me!